app/ssr.js

```diff
@@ -80,7 +93,11 @@ const {handler} = runtime.createHandler(options, (app) => {
     app.get('/favicon.ico', runtime.serveStaticFile('static/ico/favicon.ico'))
 
     app.get('/worker.js(.map)?', runtime.serveServiceWorker)
-    app.get('*', runtime.render)
+
+    app.get('*', async (req, res) => {
+        console.log(`\n\n\nreq url: ${req.url}`)
```

Comment:
This should probably be reverted.

Reply:
Yes, this was probably for debug from Dave